content/home.ts

```diff
@@ -117,6 +129,11 @@ const content: HomeContent = {
       summary: 'Does vacating a conviction make it disappear from my record?',
       details: 'No.\n\nYour conviction will still be in court records and computerized court indexes to court records. If a conviction was a domestic violence case, these records and indexes will still show the case type.\n\nInformation about the court records from the cases that led to the convictions are still public. You can still find them on [www.courts.wa.gov](https://www.courts.wa.gov/)\n\nProsecutors can still use evidence of vacated convictions in a later criminal prosecution. They can still use them in a sexually violent predator commitment proceeding.\n\nFBI records and private background check records may still have information about the convictions.',
     },
+    {
```

Comment:
Will you move this FAQ to the top of the list? On the design it looks like it should be first. I think you can bypass changing all of the ids for now by making it faq0.

Reply:
Updated